### PR TITLE
feat(GODT-2201): IMAP Empty Value Commands

### PR DESCRIPTION
**Pull request by LBeernaertProton**
Implements the following IMAP commands which have no attached values:

* Capability
* Check
* Close
* Done
* Expunge
* Idle
* Logout
* Noop
* Unselect

### CODE REVIEW

File: imap/command/done_test.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"bytes"
+	"github.com/ProtonMail/gluon/imap/parser"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestParser_DoneCommand(t *testing.T) {
+	input := toIMAPLine(`tag DONE`)
```
Comment (jameshoulahan) [inline imap/command/done_test.go:11]: done is sent without a tag

File: imap/command/logout.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"fmt"
+	"github.com/ProtonMail/gluon/imap/parser"
+)
+
+type LogoutCommand struct{}
+
+func (l LogoutCommand) String() string {
+	return fmt.Sprintf("NOOP")
```
Comment (jameshoulahan) [inline imap/command/logout.go:11]: logout

Comment (cuthix) [reply imap/command/logout.go:11]: how could it pass the test ?

File: imap/command/done_test.go (hunk 1 above)
Comment (LBeernaertProton) [reply imap/command/done_test.go]: Good catch. Done (no pun intented)

File: imap/command/logout.go (hunk 2 above)
Comment (LBeernaertProton) [reply imap/command/logout.go]: This is just for debug logging, not for the actual parsing.

Fixed.

**Review by jameshoulahan — APPROVED** [review]

**Pull request merged**